src/app/modules/lesson/lesson.component.html

```diff
@@ -4,6 +4,7 @@
       <div class="top-wrap">
         <div>
           <h1>{{ lesson.lessonTitle }}</h1>
+          <h2>{{ words.writtenBy }} {{ lesson.instructor }}</h2>
```

Comment:
My bad - since we haven't added an author for all the other lessons yet (I'll do that sometime soonish), we need  to guard for the case when we don't have an instructor, like so:
```suggestion
          <h2 *ngIf="lesson.instructor">{{ words.writtenBy }} {{ lesson.instructor }}</h2>
```
Otherwise it will look like this:
<img width="933" alt="Screenshot 2021-03-11 at 21 06 53" src="https://user-images.githubusercontent.com/16192907/110848035-e5b92000-82ad-11eb-8d40-a70c26a25064.png">